docs/source/development/getting-started.rst

```diff
@@ -0,0 +1,106 @@
+Getting Started
+===============
+
+We’re pleased that you are interested in working on Junction.
+
+This document is meant to get you setup to work on Junction and to act as a
+guide and reference to the the development setup. If you face any issues during
+this process, please `open an issue`_ about it on the issue tracker.
+
+
+Development environment setup
+*****************************
+
+You should create a virtual environment to isolate your development environment
+from the system, when developing Junction. This is usually done by creating
```

Comment:
Hi @pradyunsg, Can we start this sentence with `When developing Junction you should create..`, Instead of putting it in the end?